hal/nxp_t1024.c

```diff
@@ -2558,12 +2558,24 @@ static int hal_phy_init(struct phy_device *phydev)
 }
 
 
+#ifndef RGMII_PHY1_ADDR
 #define RGMII_PHY1_ADDR        0x4
+#endif
+#ifndef RGMII_PHY2_ADDR
 #define RGMII_PHY2_ADDR        0x3
-#define SGMII_PHY2_ADDR        0x1
-#define SGMII_PHY1_ADDR        0x2
+#endif
+#ifndef SGMII_PHY2_ADDR
```

Comment:
The PHY address is swapped here on purpose to resolve a mixup between ethernet devices.